subsys/usb/usb_descriptor.c

```diff
@@ -647,7 +664,7 @@ static struct dev_common_descriptor common_desc = {
 			.bDescriptorType = USB_INTERFACE_DESC,
 			.bInterfaceNumber = FIRST_IFACE_CDC_EEM,
 			.bAlternateSetting = 0,
-			.bNumEndpoints = 2,
+			.bNumEndpoints = NUMOF_ENDPOINTS_HID,
```

Comment:
Looks like this `NUMOF_ENDPOINTS_HID` assignment is in the wrong place.
Shouldn't this instead be under `if0` of HID?

Reply:
Thanks for pointing it out my bad. I will correct it.